apps/web/lib/webhooks/sendPayload.tsx

```diff
@@ -4,8 +4,51 @@ import { compile } from "handlebars";
 
 import type { CalendarEvent } from "@calcom/types/Calendar";
 
+import { integrationLocationToString } from "@lib/linkValueToString";
+
 type ContentType = "application/json" | "application/x-www-form-urlencoded";
 
+export type EventTypeInfo = {
+  eventTitle?: string | null;
+  eventDescription?: string | null;
+  requiresConfirmation?: boolean | null;
+  price?: number | null;
+  currency?: string | null;
+  length?: number | null;
+};
+
+function getZapierPayload(data: CalendarEvent & EventTypeInfo & { status?: string }): string {
+  const attendees = data.attendees.map((attendee) => {
+    return {
+      name: attendee.name,
+      email: attendee.email,
+      timeZone: attendee.timeZone,
+    };
+  });
+
+  const location = integrationLocationToString(data.location || "");
```

Comment:
maybe instead we add `data.locationName`

Reply:
.. I believe there is a way to return the name without the switch

Reply:
updated it to use the new `getHumanReadableLocationValue` function